core/installation/views/includes/site_initialize.php

```diff
@@ -670,7 +670,8 @@
 	'Statistics' => 1
 ));
 
-$config_path = Config::get('core/path');
+$config_path = $conf['core']['path'];
```

Comment:
This should be equivalent to `Config::get('core/path')` - does the existing code not work?

Reply:
O yeah, the problem was just that there wasn't a slash before the path.